src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericTable.js

```diff
@@ -492,7 +492,7 @@ export const GenericTable = ({
         labels={tableLabels}
         tooltipText={t(TranslationUtils.getParameterTooltipTranslationKey(parameterData.id), '')}
         dateFormat={dateFormat}
-        editMode={context.editMode}
+        editMode={!!context.editMode}
```

Comment:
probably a typo: !! (or you are using a conversion method stackoverflow didn't recommended already 13 years ago :rofl:)